frappe/email/doctype/email_queue/test_email_queue.py

```diff
@@ -39,3 +40,38 @@ def test_email_queue_deletion_based_on_modified_date(self):
 
 		self.assertTrue(frappe.db.exists("Email Queue", new_record.name))
 		self.assertTrue(frappe.db.exists("Email Queue Recipient", {"parent": new_record.name}))
+
+	def test_failed_email_notification(self):
+		subject = frappe.generate_hash()
+		email_record = frappe.new_doc("Email Queue")
+		email_record.sender = "Test <test@example.com>"
+		email_record.message = f"""MIME-Version: 1.0
+Message-Id: {frappe.generate_hash()}
+X-Original-From: Test <test@example.com>
+Subject: {subject}
+From: Test <test@example.com>
+To: <!--recipient-->
+Date: {frappe.utils.now_datetime().strftime('%a, %d %b %Y %H:%M:%S %z')}
+Reply-To: test@example.com
+X-Frappe-Site: {frappe.local.site}
+"""
```

Comment:
use `textwrap.indent` to avoid writing indented strings like this.

Reply:
Didn't understand the usage of it here, did you mean to indent the string and use textwrap.dedent()?

Reply:
![image](https://github.com/frappe/frappe/assets/10119037/2bcb1637-c0a6-40dd-a578-d2a49058066e)

> Didn't understand the usage of it here, did you mean to indent the string and use textwrap.dedent()?

Something like this

Reply:
Yeah, I meant use dedent.